### ISSUE TITLE
TypeError when using "orange"

### ISSUE THREAD

**Issue by alechp** [servexyz/tacker#10]
Screenshots & details in this PDF:

[Tacker Bug - type i is not a function.pdf](https://github.com/servexyz/tacker/files/3469533/Tacker.Bug.-.type.i.is.not.a.function.pdf)

Another protection against weird errors like this is to implement #2

**Comment by alechp** [comment]
Just checked and it looks like orange is included. 

```js
export function getChalkColor(szColor) {
  let color = szColor.toLowerCase();
  switch (color) {
    case "red":
      return chalk.red;
    case "orange":
      return chalk.orange;
...
```

However, orange isn't actually defined in Chalk's colors. You can see an example of how they're using orange [here](https://www.npmjs.com/package/chalk#256-and-truecolor-color-support)

You can see [here](https://www.npmjs.com/package/chalk#colors) that orange isn't one of the core colors

**Comment by alechp** [comment]
solved via #11
